File: Python_Codes/6. Collections Framework/Collections/Arraylist.py
class StudentList:
    # Method to add a student to the list
    def add_student(self, students, student):
        students.append(student)

    # Method to remove a student from the list
    def remove_student(self, students, student):
        if student in students:
            students.remove(student)

    # Method to print the list of students
    def print_students(self, students):
        print("Student List:", students)


def main():
    student_list = StudentList()

    # Create a list to store student names
    students = []

    # Add students to the list
    student_list.add_student(students, "Alice")
    student_list.add_student(students, "Bob")
    student_list.add_student(students, "Charlie")

    # Print the list of students
    student_list.print_students(students)

    # Remove a student from the list
    student_list.remove_student(students, "Bob")

    # Print the list of students after removal
    student_list.print_students(students)


if __name__ == "__main__":
    main()

File: Java_Codes/6. Collections Framework/Collections/Arraylist.java
package Collections;

import java.util.ArrayList;

public class Arraylist {
    // Method to add a student to the list
    public void addStudent(ArrayList<String> students, String student) {
        students.add(student);
    }

    // Method to remove a student from the list
    public void removeStudent(ArrayList<String> students, String student) {
        students.remove(student);
    }

    // Method to print the list of students
    public void printStudents(ArrayList<String> students) {
        System.out.println("Student List: " + students);
    }

    public static void main(String[] args) {
        Arraylist studentList = new Arraylist();

        // Create an ArrayList to store student names
        ArrayList<String> students = new ArrayList<>();

        // Add students to the list
        studentList.addStudent(students, "Alice");
        studentList.addStudent(students, "Bob");
        studentList.addStudent(students, "Charlie");

        // Print the list of students
        studentList.printStudents(students);

        // Remove a student from the list
        studentList.removeStudent(students, "Bob");

        // Print the list of students after removal
        studentList.printStudents(students);
    }
}
